packages/apps/job-launcher/server/src/common/constants/network.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  BINANCE_NETWORK,
+  BINANCE_TESTNET_NETWORK,
+  GOERLI_NETWORK,
+  MUMBAI_NETWORK,
+  POLYGON_NETWORK,
+} from 'nestjs-ethers';
```

Comment:
Remove these vars and add ChainId manually, we should remove all dependencies to this package